get-shit-done/workflows/update.md

```
@@ -269,15 +269,25 @@ Proceed to install step (treat as version 0.0.0 for comparison).
-**If npm check fails:**
+`LATEST_RESULT` is a JSON document with the documented shape `{ ok: bool, version: string, reason: string, detail?: string }`. Parse via `jq`:
+
+```bash
+LATEST_OK="$(printf '%s' "$LATEST_RESULT" | jq -r '.ok // false')"
+LATEST_VERSION="$(printf '%s' "$LATEST_RESULT" | jq -r '.version // empty')"
+LATEST_REASON="$(printf '%s' "$LATEST_RESULT" | jq -r '.reason // empty')"
+```
+
+**If `LATEST_OK` is not `true`** (or `LATEST_STATUS` is non-zero):
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)